src/test/java/org/opentripplanner/updater/trip/TimetableSnapshotSourceTest.java

```diff
@@ -1052,14 +1164,162 @@ public void repeatedlyAddedTripWithNewRoute() {
         List.of(tripUpdate),
         feedId
       );
-      var secondPattern = assertAddedTrip(SERVICE_DATE, addedTripId, updater);
+      var secondPattern = assertAddedTrip(SERVICE_DATE, addedTripId, updater, false);
       var secondRoute = secondPattern.getRoute();
 
       // THEN
 
       assertSame(firstRoute, secondRoute);
       assertNotNull(transitModel.getTransitModelIndex().getRouteForId(firstRoute.getId()));
     }
+
+    @Test
+    public void cancelingAddedTrip() {
+      // TODO we might want to change the behaviour so that only the trip without pattern is
```

Comment:
To me it looks like those two tests do almost exactly the same. Can they be a parameterized test?

Reply:
In theory yes, if I do that, where should I put the TODO if anywhere?

Reply:
You can be creative. :)

Maybe on the definition of the test case?

Reply:
Done